### PR TITLE
Fixes crash on specific, small zones 

### PR DESCRIPTION
**Pull request by madsnedergaard**
### Description

After the new geojson setup, clicking on some zones crashes the app: https://app.electricitymap.org/zone/US-SW-SRP
Screenshot 2022-01-11 at 10 21 53

This happens due to the structure we're expecting, which apparently now may differ depending on the zone. I've only experienced the issue on certain US zones, but hopefully this will catch all of these errors :)

### CODE REVIEW

File: web/src/helpers/map.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { max, min, mean } from 'lodash';
+import { max, min, mean, isArray } from 'lodash';
 
-export function getCenteredLocationViewport([longitude, latitude]) {
+export function getCenteredLocationViewport([longitude, latitude]) {''
```
Comment (skovhus) [inline web/src/helpers/map.js:3]: This looks a bit funky... `''`

**Review by skovhus — CHANGES_REQUESTED** [review]

**Review by pierresegonne — APPROVED** [review]
Is is normal that the format is not consistent? 
I guess it's because now a zone can consist of several polygons? 

LGTM anyhow

File: web/src/helpers/map.js (hunk 1 above)
Comment (skovhus) [reply web/src/helpers/map.js:3]: Why is the linter not catching this?

Comment (madsnedergaard) [reply web/src/helpers/map.js]: Dunno, but thanks for catching it! Fixed now

**Review by skovhus — APPROVED** [review]

**Comment by skovhus** [comment]
I'm pretty sure this is caused by the arc logic here that is specific to US: https://github.com/electricitymap/electricitymap-contrib/blob/master/web/src/helpers/topos.js#L10

**Pull request merged**